### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by Juanse7793**
## Requirements
- Diagram:

- We create a file named schema_based_on_diagram.sql where we implement the database from the diagram.
- We create join tables from many-to-many relationships.
- We add the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  type VARCHAR(255),
+  name VARCHAR(255),
+);
+
+ALTER TABLE medical_histories ADD CONSTRAINT fk_patient FOREIGN KEY (patient_id) REFERENCES patients (id);
+ALTER TABLE invoice_items ADD CONSTRAINT fk_invoice FOREIGN KEY (invoice_id) REFERENCES invoices (id);
+ALTER TABLE invoices ADD CONSTRAINT fk_med_his FOREIGN KEY (medical_history_id) REFERENCES medical_histories (id);
+ALTER TABLE treatments ADD CONSTRAINT fk_id FOREIGN KEY (id) REFERENCES medical_histories (id);
+ALTER TABLE invoice_items ADD CONSTRAINT fk_treatment FOREIGN KEY (treatment_id) REFERENCES treatments (id);
+
```
Comment (brytebee) [inline schema_based_on_diagram.sql:42]: - [ ] **Required:** Great job coming this far! 👏 
However, you have not created the indexes. PostgreSQL automatically creates indexes on primary keys and unique constraints, but not on the referencing side of foreign key relationships. Kindly explicitly create indexes for your FK's.

_**Hint**_ Consider the code below 

```
CREATE INDEX patient_id_asc 
  ON medical_histories
  (patient_id ASC);
  ```

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Juanse7793 & @Alexr16 & @Dangelo-JAN 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- All tables from the diagram are created.✔️
- All tables have a PRIMARY KEY set.✔️
- All FOREIGN KEYS are set.✔️
- All "join table"s for the many-to-many relationships are created.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️

- Kindly update your `README.md` to reflect all contributors to the project.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (Alexr16) [reply schema_based_on_diagram.sql:42]: @brytebee Thanks for your feedback. The requirement is done.

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**